lib/deaf_grandma.rb

```diff
@@ -19,8 +19,21 @@ def run!
 
 
   def speak(input)
```

Comment:
Nice job solving it!  Does this method look like it's got a lot of decisions being made?  Think about how you might be able to refactor this to make the code paths a little easier to follow.

Reply:
Refactored the code to reduce if statements and minimize code block length